content/docs/docs-nav.yaml

```diff
@@ -39,8 +39,8 @@ nav:
   - Custom plugins:
     - Artifactory: '/docs/custom-plugins/artifactory/'
     - Configuring: '/docs/custom-plugins/configuring/'
-    - Authenticated Proxy: '/docs/custom-plugins/authenticated-proxy/'
   - Details:
-    - GitHub app permissions: '/docs/integrations/github-app-permissions/'
+    - GitHub app permissions: '/docs/details/github-app-permissions/'
```

Comment:
I think you should leave this as "integrations". If you change this, anything which links to this page will start 404ing. If you are confident that nothing links here it might be ok to move it.

Reply:
Thanks. I just updated the PR to fix the relative link. It was only in one place!

Reply:
I think we should be good to go now